tests/product/test_configuration.py

```diff
@@ -337,16 +317,7 @@ def test_configuration_show_coord_worker_using_dash_x(self):
             expected = f.read()
         self.assertRegexpMatches(output, expected)
 
-    def test_configuration_no_presto_user(self):
```

Comment:
I actually think this is valuable, as it makes sure that our files end up with the right permissions.  I'm going to add this test back.  I'll add some unit tests when I fix #266.  Maybe we can revisit then and see if those are sufficient.